packages/frontapp/src/app/components/AllBooksList/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { FC } from 'react';
+import BookCard from '../BookCard';
+import styled from '@emotion/styled';
+import { dimensions } from '@mimir/ui-kit';
+const WrapperList = styled.section`
+  display: flex;
+  gap: ${dimensions.base_2};
+  width: 100%;
+  flex-wrap: nowrap;
+  max-height: 22rem;
+  overflow-x: hidden;
+`;
+
+export interface IMaterial {
+  id: string;
+  __typename?: 'Material';
+  picture?: string | null | undefined;
+  category: string;
+  author: string;
+  title: string;
+  created_at: string;
+}
```

Comment:
You could just export this type from global-types.ts

Reply:
it doesnt work, and requires a lot of additional attributes..

Reply:
@sabrina34352 Sorry, from client-types.ts in apollo-client lib. https://www.typescriptlang.org/docs/handbook/utility-types.html#picktype-keys

Reply:
@x8er yes i did it, thank you for pointing out